modules/violation-view-policy-tab.adoc

```diff
@@ -9,21 +9,36 @@
 The *Policy* tab of the *Details* panel displays details of the policy that caused the violation.
 
 [discrete]
-== Policy Details section
+== Policy overview section
 
-The policy details section lists the following information:
+The policy overview section lists the following information:
 
-* *Id*: The numerical identifier for the policy.
-* *Name*: The name of the policy.
+* *Severity*: A ranking of the policy (critical, high, medium, or low) for the amount of attention required.
+* *Categories*: The policy category of the policy. Policy categories are listed in *Platform Configuration* -> *Policy Management* in the *Policy categories* tab.
+* *Type*: Whether the policy is user generated (policies created by a user) or a system policy (policies built into {product-title-short} by default).
 * *Description*: A detailed explanation of what the policy alert is about.
 * *Rationale*: Information about the reasoning behind the establishment of the policy and why it matters.
-* *Remediation*: Suggestions on how to fix the violation.
-* *Enabled*: Indicates if the policy is enabled.
-* *Categories*: The policy category of the policy.
+* *Guidance*: Suggestions on how to address the violation.
+* *MITRE ATT&CK*: Indicates if there are MITRE link:https://attack.mitre.org/matrices/enterprise/containers/[tactics and techniques] that apply to this policy.
+
+[discrete]
+== Policy behavior
+
+The policy behavior section provides the following information:
+
 * *Lifecycle Stage*: Lifecycle stages that the policy belongs to, `Build`, `Deploy`, or `Runtime`.
-* *Severity* - The risk level for the violation.
+* *Event source*: This field is only applicable if the lifecycle stage is `Runtime`. It can be one of the following:
+** *Deployment*: {product-title-short} triggers policy violations when event sources include process and network activity, pod exec, and pod port forwarding.
+** *Audit logs*: {product-title-short} triggers policy violations when event sources match Kubernetes audit log records.
+* *Response*: The response can be one of the following:
+** *Inform*: Policy violations generate a violation in the violations list.
+** *Inform and enforce*: The violation is enforced.
+* *Enforcement*: If the response is set to *Inform and enforce*, lists the type of enforcement that is set for the following stages:
+** *Build* {product-title-short} fails your continuous integration (CI) builds when images match the criteria of the policy.
+** *Deploy* - {product-title-short} blocks creation of deployments that match the conditions of the policy. In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments. In other clusters, {product-title-short} edits noncompliant deployments to prevent pods from being scheduled.
```

Comment:
```suggestion
** *Deploy*: {product-title-short} blocks creation of deployments that match the conditions of the policy. In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments. In other clusters, {product-title-short} edits noncompliant deployments to prevent pods from being scheduled.
```